resources/lang/de.json

```diff
@@ -65,6 +65,8 @@
     "controller.user.follow-delete-not-permitted": "Diese Aktion ist nicht erlaubt.",
     "controller.user.follow-destroyed": "Du folgst dieser Person nicht mehr.",
     "controller.user.follow-ok": "Benutzer gefolgt.",
+    "controller.user.follow-request-already-exists": "Du hast dieser Person bereits ein Folgen angefragt.",
+    "controller.user.follow-request-ok": "Folgen angefragt.",
```

Comment:
same here...

"Du hast :username angefragt zu folgen."?

Reply:
Idk, i like it.